src/Chat.tsx

```diff
@@ -176,10 +179,14 @@ export class Chat extends React.Component<ChatProps, {}> {
 
         return (
             <Provider store={ this.store }>
+
                 <div className="wc-chatview-panel" ref={ div => this.chatviewPanel = div }>
                     { header }
                     <MessagePane setFocus={ () => this.setFocus() }>
-                        <History setFocus={ () => this.setFocus() }/>
+                        <History
+                          setFocusOnCardActionClick={this.props.setFocusOnCardActionClick}
```

Comment:
If we add the conditional here, we won't need the additional property on History